app/views/locations/index.html.erb

```diff
@@ -63,7 +63,7 @@
           }
     %>
 
-    <p>Displaying results near <b><%= @search.query %></b><%= " (page #{@locations.current_page} of #{@locations.total_pages})" if @locations.total_pages.positive? %>.</p>
+    <p>Displaying results near <b><span id="search-query"><%= @search.query %></span></b><%= " (page #{@locations.current_page} of #{@locations.total_pages})" if @locations.total_pages.positive? %>.</p>
```

Comment:
The addition of the `<span>` doesn't do anything that couldn't be done by adding the `id` attribute to the `<b>` element.

Also, you _might_ be able to do `assert_select 'b', text: 'matching text goes here'` in the spec and avoid the `id` attribute. That's a guess, though.